### DISCUSSION TITLE
Extend System.TypeCode and System.Convert with new intrinstic data types

### DISCUSSION THREAD

**Discussion by SetTrend** [dotnet/roslyn#74898] [Ideas] [answered]
Are there any plans for extending [`System.TypeCode`](https://learn.microsoft.com/en-us/dotnet/api/system.typecode) (and, thus, [`System.Convert`](https://learn.microsoft.com/en-us/dotnet/api/system.convert)) with the new intrinsic base data types, like, e.g., [`System.DateOnly`](https://learn.microsoft.com/en-us/dotnet/api/system.dateonly)?

**Comment by CyrusNajmabadi** [comment, marked as answer]
You should ask that if dotnet/runtime :-)

Unfortunately, I cannot move this question there.

**Comment by 333fred** [comment]
This is a question for https://github.com/dotnet/runtime/.

**Comment by SetTrend** [comment]
Thanks for pointing me! :)
